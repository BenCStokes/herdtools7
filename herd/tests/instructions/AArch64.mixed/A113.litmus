AArch64 A113
(* strh immediate, with offset, symbolic location *)

Variant=mixed

 { 0:x0=1; 0:x2=x; uint64_t x;}

P0;
  STRH W0, [X2, #4];

forall (x=0)
