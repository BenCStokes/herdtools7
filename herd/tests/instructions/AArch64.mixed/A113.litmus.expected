Test A113 Required
States 1
[x]=0x100000000;
No
Witnesses
Positive: 0 Negative: 1
Condition forall ([x]=0x0)
Observation A113 Never 0 1
Hash=4004151daf47c1c13517281d9010dc3e

